apps/1kv-backend/values.yaml

```diff
@@ -13,13 +15,55 @@ kusama:
     domain: w3f.community
 polkadot: 
   db:
+    architecture: replicaset
     uri: "mongodb://<path:vaults/k8s-community-secrets/items/otv-polkadot#mongo-username>:<path:vaults/k8s-community-secrets/items/otv-polkadot#mongo-password>@polkadot-mongodb-headless.polkadot.svc.cluster.local/otv?tls=false&replicaSet=rs0"
   be:
+    nominating: true
     secret:  <path:vaults/k8s-community-secrets/items/otv-polkadot#backend-secret>
     domain: polkadot.w3f.community
     matrix:   
       room: "!ccXWsugXuoiHnkEhKa:web3.foundation"
       userId: "@1kv-stats:matrix.org"
   mongoExpress:
     whitelistSourceIps: <path:vaults/k8s-community-secrets/items/otv-polkadot#shared-vpn-ip>
-    domain: w3f.community      
\ No newline at end of file
+    domain: w3f.community
+
+resources:
+  kusama:
+    db:
+      storageSize: "8Gi"
+    resources:
+      limits: 
+        cpu: "2000m"
+        memory: "2Gi"
+      requests: 
+        cpu: "1000m"
+        memory: "1Gi"
+    be:
+      storage: "20Gi"
+      resources:
+        limits:
+          cpu: "3500m"
+          memory: "9Gi"
+        requests:
+          cpu: "700m"
+          memory: "3Gi"
+  polkadot:
+    db:
+      storageSize: "8Gi"
+    resources:
+      limits: 
+        cpu: "1200m"
+        memory: "2Gi"
+      requests: 
+        cpu: "1000m"
+        memory: "1Gi"
+    be:
+      storage: "20Gi"
```

Comment:
keep it consistent, either storage or storageSize

Reply:
ok